pinot-broker/src/main/java/org/apache/pinot/broker/api/resources/PinotClientRequest.java

```diff
@@ -121,6 +121,8 @@ public void processSqlQueryGet(@ApiParam(value = "Query", required = true) @Quer
       }
       BrokerResponse brokerResponse = executeSqlQuery(requestJson, makeHttpIdentity(requestContext), true);
       asyncResponse.resume(brokerResponse.toJsonString());
+    } catch (WebApplicationException wae) {
```

Comment:
Put some error log? Also do we have a metric tracking the auth failure?
Same for other places

Reply:
Added log. There is a Meter metric REQUEST_DROPPED_DUE_TO_ACCESS_ERROR("requestsDropped") which is tracking at several places, but let me file a task to review and add at the missing places, such as request filter.